src/components/Maps/Marker.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import RoomIcon from '@material-ui/icons/Room';
+import RadioButtonCheckedIcon from '@material-ui/icons/RadioButtonChecked';
+import { Colours } from '../../styles/Constants';
+
+const useStyles = makeStyles({
+  root: {
+    color: Colours.Marker,
+  },
+  currentLocation: {
+    color: Colours.CurrentLocationMarker,
+  },
+  notClicked: {
+    color: Colours.MarkerNotClicked,
+  },
+});
+
+const Marker = ({
+  /* eslint-disable */
```

Comment:
I need to include this to avoid linting errors. We need to pass in lat and lng because the API wrapper that we are using right now depends on us passing latitudes and longitudes to any child components of the map

Reply:
what's the linter error? seems kinda strange that we only want to exclude these 2 lines

Reply:
Linter complains that the variables are unused, but we need to pass it in